### ISSUE TITLE
Unable to download NYU depth dataset

### ISSUE THREAD

**Issue by JoeCameron98** [ethz-asl/background_foreground_segmentation#32]
I'm attempting to download the NYU depth dataset:
http://horatio.cs.nyu.edu/mit/silberman/nyu_depth_v2/nyu_depth_v2_labeled.mat

And I'm getting "ERR_CONNECTION_REFUSED"

I similarly get a connection failure when running the data_setup.py script.

I have checked with our Network Administrator and they say there is nothing blocking it on our side.

Is the data still stored at this location? Or has it moved elsewhere? I would like to investigate training on this data so having access to it would be awesome.

Thanks in advance,

Joe

**Comment by hermannsblum** [comment]
It looks like it has moved, can you try the download link from this website? https://cs.nyu.edu/~silberman/datasets/nyu_depth_v2.html

**Comment by JoeCameron98** [comment]
Hey, I tried using the link you sent and I had the same issue with the connection being refused.

Is the NYU depth dataset available elsewhere?

**Comment by hermannsblum** [comment]
You are right, this is unfortunate. You will have to look for mirrors. Did you already try this one?
https://huggingface.co/datasets/sayakpaul/nyu_depth_v2

If you write me an email I can also check whether I have it still stored somewhere. I can try to share this via email, but don't think I am allowed to publicly re-host the dataset.

**Comment by JoeCameron98** [comment]
Hey, 

Thanks for your reply, I am yet to try the huggingface link (I'm away from my computer) but the data downloading script there references the same web address as is given in this repo so I can't imagine it will work.

Please message me privately and we can talk about accessing the dataset via email. I think it should be fine to share as the data is under Apache 2.0 license

**Comment by hermannsblum** [comment]
please check the link properly, it references the link in the readme but the data is mirrored and hosted on huggingface
